src/docs/spec-model.md

```diff
@@ -92,6 +92,25 @@ In addition, predicates from the following sources MAY also be encouraged:
 * any relation under [skos:mappingRelation](http://www.w3.org/2004/02/skos/core#mappingRelation) in the [Semantic Mapping Vocabulary](https://mapping-commons.github.io/semantic-mapping-vocabulary/).
 
 
+## Literal mappings
+
+<a id="literal-mappings"></a>
+
+The SSSOM model is primarily intended to represent mappings between semantic entities. However, it may also be used to represent mappings where one side is a literal string that does not have an identifier of its own. Any such mapping is henceforth called a _literal mapping_.
```

Comment:
"where one side" - If we want to specifically prohibit literal - literal mappings we should probably make a very clear in the spec, and add this to the schema as a separate postcondition (if subject_type literal --> subject_label required AND object_if required)?

Reply:
Do we want to forbid literal-to-literal mappings? I have no objection to that, but I see no reason to do it, either.

If we do, I wouldn’t bother expressing that as a LinkML postcondition – they are useless as far as running code is concerned.

Reply:
I would personally not forbid it, for no other reasons other than that there is no strong _technical_ reason to do so. Literal to literal mappings may sound useless now, but I can see some use cases for them like data model mappings (where enums are represented as strings). So my inclination would be to simply change the phrasing `where one side is a literal string` to `where at least one side is a literal string`.

Reply:
> So my inclination would be to simply change the phrasing where one side is a literal string to where at least one side is a literal string.

OK, will do.